projects/pgai/tests/vectorizer/cli/test_vectorizer_document.py

```diff
@@ -175,6 +181,12 @@ def test_simple_document_embedding_local(
         # lego_sets.pdf
         assert "7190-1 Millennium Falcon" in chunks_str
 
+        # text_screenshot.png
+        assert "Use cases include providing chatbot" in chunks_str
+
+        # electromagnetic_radiation.docx
+        assert "All forms of EMR travel at the speed of light in a vacuum" in chunks_str
```

Comment:
We can remove this test and save some KB but I added it in order to prove no extra models were being downloaded during the parsing.